app/main.py

```diff
@@ -1,8 +1,69 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int | float,
+            clean_power: int,
+            average_rating: int | float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        """
+        The method, that takes a list of cars, washes only cars with clean_mark < clean_power of
+        wash station and returns income of CarWashStation for serving this list of cars,
+        rounded to 1 decimal.
+        """
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        """
+        The method, that calculates cost for a single car wash, cost is calculated as: car's comfort class
+        * difference between wash station's clean power and car's clean mark
+        * car wash station rating / car wash station distance to the center of the city.
+        """
+        price = (
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * (self.average_rating
+               / self.distance_from_city_center))
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> int:
+        """
+        The method, that washes a single car, so it should have clean_mark equals wash station's clean_power,
+        if wash_station.clean_power is greater than car.clean_mark.
+        """
+        car.clean_mark = self.clean_power
+        return car.clean_mark
```

Comment:
Code Quality Issue: The return statement in the wash_single_car method is unnecessary. The method description states that it should wash a single car, so it should have clean_mark equals wash station's clean_power, if wash_station.clean_power is greater than car.clean_mark. However, it does not mention about returning anything. So, consider removing the return statement. This is in line with the first point in the Clean Code section of the checklist.